contracts-periphery/test/utils/DSTestPlus.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity ^0.8.13;
 
 import "forge-std/Test.sol";
+import "forge-std/Vm.sol";
```

Comment:
I don't think this import is needed since `Test.sol` has a `Vm` instance and we inherit from `Test`?

Reply:
You're right!